src/include/cell.i

```diff
@@ -685,6 +700,7 @@ restart:
     unpack->start_txn = WT_TXN_NONE;
     unpack->stop_ts = WT_TS_MAX;
     unpack->stop_txn = WT_TXN_MAX;
+    unpack->oldest_durable_ts = WT_TS_NONE;
     unpack->newest_durable_ts = WT_TS_NONE;
```

Comment:
newest_durable_ts to be initialized with WT_TS_MAX.

Reply:
Fixed.